.circleci/config.yml

```diff
@@ -35,6 +35,11 @@ jobs:
             - pl-npm-v3-{{ checksum "patternlab/styleguide/package-lock.json" }}
             - pl-npm-v3-
       - run: { name: 'NPM Install', command: 'cd patternlab/styleguide && npm install' }
+      - run: { name: 'Yarn Install', command: 'yarn install' }
```

Comment:
See the comment in dangerfile.js. Move these steps into a new job and filter on branch in workflows